overlord/ifacestate/handlers.go

```diff
@@ -1217,3 +1217,48 @@ func (m *InterfaceManager) doGadgetConnect(task *state.Task, _ *tomb.Tomb) error
 	task.SetStatus(state.DoneStatus)
 	return nil
 }
+
+// doHotplugSeqWait returns Retry error if there is another change for same hotplug key and a lower sequence number.
+// The handler expects "hotplug-key" and "hotplug-seq" values set on own and other hotplug-related changes.
```

Comment:
Can you add a comment about the idea behind using the sequence number. How the use of a sequence number solves a problem that was there before.

Reply:
Added.